docs/how-to/native-install/install-faq.rst

```diff
@@ -91,3 +91,44 @@ The returned information should reflect the addition of ``iommu``:
 
 Refer to `RCCL Issue #1129 <https://github.com/ROCm/rccl/issues/1129>`_ for more information. 
 
+.. _troubleshooting-install-missing-packages-for-dockers:
+
+Issue #6: Additional packages for dockers
+================================================
+
+Docker containers may not include some basic packages.  When installing ROCm within docker containers, you may need to install packages prior to following the ROCm install instructions. Here are some suggested steps when using docker containers.
+
+.. tab-set::
+
+  .. tab-item:: Ubuntu
+
+    .. code-block:: shell
+
+      apt update
+      apt install sudo wget
+
+
+  .. tab-item:: RHEL
+
+    .. code-block:: shell
+
+      dnf install sudo wget
+      subscription-manager register --username <username> --password <password>
+      subscription-manager attach --auto
+      subscription-manager repos --enable codeready-builder-for-rhel-9-x86_64-rpms

```

Comment:
```suggestion
      subscription-manager register --username <username> --password <password>
      subscription-manager attach --auto
      subscription-manager repos --enable codeready-builder-for-rhel-9-x86_64-rpms

```

Maybe we can remove from this part because we talk about registering in the next section

Reply:
I think we'll leave this part for now.  I know it's a bit redundant, but I just want it covered for now.